### PR TITLE
Update eval suite

### PR DESCRIPTION
**Pull request by hamishivi**
Updating the eval suite. For tulu 3 evals, we now entirely rely on the oe-eval suite and nouhas safety eval fork (which hopefully will be added to the oe-eval tool soon). This means that we need an option to skip running the safey evals.

Also adding a script to upload metadata in case someone wants to update or add it post-hoc.

### CODE REVIEW

File: scripts/add_metadata.py
Comment (natolambert) [inline scripts/add_metadata.py:1]: Nice!

**Review by natolambert — APPROVED** [review]
Looks like what I would expect and solid.

**Review by vwxyzjn — APPROVED** [review]
Nice change! Could you add `--skip_oi_evals` for the auto eval as well?


https://github.com/allenai/open-instruct/blob/903d526499cfb37eb6dad385e19efe947d72c9d0/open_instruct/utils.py#L721-L739

File: open_instruct/utils.py [hunk 1]
```diff
@@ -743,6 +745,10 @@ def submit_beaker_eval_jobs(
         command += f" --upload_to_hf {upload_to_hf}"
     if run_oe_eval_experiments:
         command += " --run_oe_eval_experiments"
+    if run_safety_evaluations:
+        command += " --run_safety_evaluations"
+    if skip_oi_evals:
```
Comment (hamishivi) [inline open_instruct/utils.py:750]: @vwxyzjn `skip_oi_evals` is added here...

File: scripts/wait_beaker_dataset_model_upload_then_evaluate_model.py [hunk 2]
```diff
@@ -39,6 +39,8 @@ def main(args: Args, beaker_runtime_config: BeakerRuntimeConfig):
                 model_name=args.model_name,
                 location=beaker_dataset_ids[0],
                 run_oe_eval_experiments=True,
+                run_safety_evaluations=True,
+                skip_oi_evals=True,
```
Comment (hamishivi) [inline scripts/wait_beaker_dataset_model_upload_then_evaluate_model.py:43]: @vwxyzjn ...and then set here.

**Pull request merged**